pkg/network/protocols/http2/protocol.go

```diff
@@ -265,17 +266,30 @@ func (p *Protocol) PostStart(mgr *manager.Manager) error {
 	return p.dynamicTable.postStart(mgr, p.cfg)
 }
 
+func getMap(mgr *manager.Manager, name string) (*ebpf.Map, error) {
+	m, _, err := mgr.GetMap(name)
+	if err != nil {
+		return nil, fmt.Errorf("error getting %q map: %s", name, err)
+	}
+	if m == nil {
+		return nil, fmt.Errorf("%q map is nil", name)
+	}
+	return m, nil
+}
+
 func (p *Protocol) updateKernelTelemetry(mgr *manager.Manager) {
-	mp, _, err := mgr.GetMap(telemetryMap)
+	mp, err := getMap(mgr, telemetryMap)
 	if err != nil {
-		log.Warnf("unable to get http2 telemetry map: %s", err)
+		log.Warn(err)
 		return
 	}
 
-	if mp == nil {
-		log.Warn("http2 telemetry map is nil")
+	tlsMap, err := getMap(mgr, tlsTelemetryMap)
+	if err != nil {
```

Comment:
maybe lets wrap the error to be able to easily understand if the issue is realted to the plain or not

Reply:
we do know, as the inner error contains the map name